_example/echo/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/arl/statsviz"
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	// Echo instance
```

Comment:
All examples do the following as the first line of `main`, I'd like to do the same here:
```go
	// Force the GC to work to make the plots "move".
	go example.Work()
```
This is just so that the plots are not static, they _move_ since the garbage collector is working.